pdart/pipeline/SuffixInfo.py

```diff
@@ -0,0 +1,420 @@
+        ["WFC3"],
+    ),
+    ("WFC3", "UVIS", "crj"): (  # because crc is also available
+        True,
+        True,
+        False,
+        "Calibrated",
+        "Data",
+        "Combined, calibrated {I} image file {F} from repeated exposures, without CTE correction, in HST Program {P}",
+        "Combined, calibrated {I} image files from repeated exposures, without CTE correction, in HST Program {P}",
+        ["WFC3"],
+    ),
+    ("WFC3", "UVIS", "drz"): (  # because drc is also available
+        True,
+        True,
+        False,
+        "Derived",
+        "Data",
+        "Calibrated {I} image file {F}, corrected for geometric distortion but not CTE, from HST Program {P}",
+        "Calibrated {I} image files, corrected for geometric distortion but not CTE, from HST Program {P}",
+        ["WFC3"],
+    ),
+    ("WFC3", "IR", "raw"): (  # Why do this?  Possibly remove.
+        True,
+        True,
+        False,
+        "Raw",
+        "Data",
+        "Raw, uncalibrated {I} image file {F}, without CR rejection, from HST Program {P}",
+        "Raw, uncalibrated {I} image files, without CR rejection, from HST Program {P}",
+        ["WFC3"],
+    ),
+    ("WFC3", "IR", "ima"): (  # Doesn't match handbook.
+        False,
+        False,
+        False,
+        "Partially Processed",
+        "Data",
+        "Calibrated {I} image file {F}, without CR rejection, from HST Program {P}",
+        "Calibrated {I} image files, without CR rejection, from HST Program {P}",
+        ["WFC3"],
+    ),
+    ("WFC3", "IR", "flt"): (  # Why do this?  Possibly remove.
+        True,
+        True,
+        False,
+        "Calibrated",
+        "Data",
+        "Calibrated {I} image file {F}, without CR rejection, from HST Program {P}",
+        "Calibrated {I} image files, without CR rejection, from HST Program {P}",
+        ["WFC3"],
+    ),
+    ("NICMOS", "cal"): (
+        True,
+        False,
+        False,
+        "Calibrated",
+        "Data",
+        "Calibrated {I} image file {F} from HST Program {P}",
+        "Calibrated {I} image files from HST Program {P}",
+        ["NICMOS"],
+    ),
+    ("NICMOS", "ima"): (
+        False,
+        False,
+        False,
+        "Partially Processed",
+        "Data",
+        "Calibrated but uncombined {I} image file {F} for MULTIACCUM observations in HST Program {P}",
+        "Calibrated but uncombined {I} image file for MULTIACCUM observations in HST Program {P}",
+        ["NICMOS"],
+    ),
+    ("NICMOS", "mos"): (
+        True,
+        False,
+        False,
+        "Derived",
+        "Data",
+        "Combined, calibrated {I} image file {F} for dithered observations in HST Program {P}",
+        "Combined, calibrated {I} image files for dithered observations in HST Program {P}",
+        ["NICMOS"],
+    ),
+}
+
+# First letter of filenames and the corresponding instrument names
+INSTRUMENTS_INFO = {
```

Comment:
A better name for this global definition would be "INSTRUMENT_FROM_LETTER_CODE" or something like that. INSTRUMENT_INFO is too generic a name for this variable. Also, we should adopt a global standard for whether to identify instrument names using upper case or lower case. It's upper case above but lower case here. (I recommend we go with upper case everywhere because that is how instrument names are usually written.)